### ISSUE TITLE
makeinfo: command not found  ...  [candl.info] Error 127  ...  [all] Error 2

### ISSUE THREAD

**Issue by wbraswell** [bondhugula/pluto#18]
I am automating the Pluto build process with my new Alien::Pluto software for Perl:

https://github.com/wbraswell/alien-pluto

However it fails to build due to a missing `makeinfo` command, which is not right because I have not changed any documentation files (or any other files whatsoever) in the Pluto tarball...

https://api.travis-ci.org/v3/job/316614841/log.txt

```
Making all in candl
make[3]: Entering directory `/home/travis/build/wbraswell/alien-pluto/_alien/build_73oc/pluto-0.11.4/candl'
Making all in doc
make[4]: Entering directory `/home/travis/build/wbraswell/alien-pluto/_alien/build_73oc/pluto-0.11.4/candl/doc'
make[5]: Entering directory `/home/travis/build/wbraswell/alien-pluto/_alien/build_73oc/pluto-0.11.4/candl/doc'
  MAKEINFO candl.info
/home/travis/build/wbraswell/alien-pluto/_alien/build_73oc/pluto-0.11.4/candl/autoconf/missing: line 81: makeinfo: command not found
WARNING: 'makeinfo' is missing on your system.
         You should only need it if you modified a '.texi' file, or
         any other file indirectly affecting the aspect of the manual.
         You might want to install the Texinfo package:
         <http://www.gnu.org/software/texinfo/>
         The spurious makeinfo call might also be the consequence of
         using a buggy 'make' (AIX, DU, IRIX), in which case you might
         want to install GNU make:
         <http://www.gnu.org/software/make/>
make[5]: *** [candl.info] Error 127
make[5]: Leaving directory `/home/travis/build/wbraswell/alien-pluto/_alien/build_73oc/pluto-0.11.4/candl/doc'
make[4]: *** [all-recursive] Error 1
make[4]: Leaving directory `/home/travis/build/wbraswell/alien-pluto/_alien/build_73oc/pluto-0.11.4/candl/doc'
make[3]: *** [all-recursive] Error 1
make[3]: Leaving directory `/home/travis/build/wbraswell/alien-pluto/_alien/build_73oc/pluto-0.11.4/candl'
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory `/home/travis/build/wbraswell/alien-pluto/_alien/build_73oc/pluto-0.11.4'
make[1]: *** [all] Error 2
make[1]: Leaving directory `/home/travis/build/wbraswell/alien-pluto/_alien/build_73oc/pluto-0.11.4'
external command failed at /home/travis/perl5/perlbrew/perls/5.10/lib/site_perl/5.10.1/Alien/Build/CommandSequence.pm line 87.
make: *** [_alien/mm/build] Error 2
...
The command "make" exited with 2.
```

I am NOT running a buggy make on Travis:

```
$ make -v
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for x86_64-pc-linux-gnu
```

The bottom line is that no part of Pluto should be trying to call `makeinfo` during the build process.

Please let me know as soon as this issue has been resolved so that I can test it again and then publish the Alien::Pluto software.

Thanks in advance!  :-)

**Comment by bondhugula** [comment]
What's the output of your
$ git submodule

Which commit of Pluto are you using?

**Comment by wbraswell** [comment]
@bondhugula 
As you can see from the Travis log output file I linked above, it is directly downloading your Pluto 0.11.4 tarball from GitHub:

```
Alien::Build::Plugin::Core::Download> candidate *https://github.com/bondhugula/pluto/files/737550/pluto-0.11.4.tar.gz
```
This is an official Pluto release, so we definitely do not expect to see the buggy `makeinfo` behavior.

**Comment by bondhugula** [comment]
> This is an official Pluto release, so we definitely do not expect to see the buggy makeinfo behavior.
Releases can have bugs too! This issue is from the candl library (a submodule of Pluto which is shipped along). It was fixed upstream in candl and was fixed in the Pluto git post the 0.11.4 release. It will be available in 0.11.5, which is long overdue, but I don't have an eta yet. In the next two months perhaps.

**Comment by wbraswell** [comment]
Oh yes I know all about bugs in releases, I was just surprised to see a major bug like this because it is causing total build failure.  :-P
I am very glad to hear this will be fixed in the next official Pluto release v0.11.5!  :-)
I was hoping to have this particular bug taken care of in the next few days instead of a few months...

**Comment by bondhugula** [comment]
BTW, the bug will go away if you have texinfo installed. (package name 'texinfo' on an Ubuntu or Fedora).

**Comment by wbraswell** [comment]
LOL yes sir I am way ahead of you, I have implemented the automated building of Texinfo in Perl as a temporary solution, but it is obviously not the final answer...
https://github.com/wbraswell/alien-texinfo
https://metacpan.org/pod/Alien::Texinfo

**Comment by wbraswell** [comment]
@bondhugula 
I see you have closed this issue, although I can not personally confirm that the issue has been fixed in production until you release v0.11.5...
